Orbitersdk/samples/ProjectApollo/src_rtccmfd/ApolloRTCCMFD.cpp

```diff
@@ -1362,6 +1441,33 @@ bool GenericIntInputBox(void *id, char *str, void *data)
 	return false;
 }
 
+void ApolloRTCCMFD::GenericVectorInput(VECTOR3 *val, char* message, double factor)
+{
+	void *data2;
+
+	tempData.vVal = val;
+	tempData.factor = factor;
+	data2 = &tempData;
+
+	bool GenericVectorInputBox(void *id, char *str, void *data);
+	oapiOpenInputBox(message, GenericVectorInputBox, 0, 25, data2);
```

Comment:
```suggestion
	oapiOpenInputBox(message, GenericVectorInputBox, 0, 25, &tempData);
```
You can just pass the address directly unless I'm missing something.